docs/operation/observability.md

```diff
@@ -24,6 +24,238 @@ When query tracing is enabled, the slow query log won't be generated.
 
 ## Metrics
 
+BanyanDB expose metrics for monitoring and analysis. In this part, we use some variables to represent the metrics, such as `$job` and `$instance`. The `$job` is the job name of the BanyanDB collection job, and the `$instance` is the instance name of the BanyanDB instance.
+
+`__rate_interval` is a variable that represents the rate interval. It is used to calculate the rate of metrics.
+
+### Stats
+
+`Stats` metrics are used to monitor the overall status of BanyanDB. The following metrics are available:
+
+#### Write Rate
+
+The write rate is the number of write operations per second. It is calculated by summing the total number of written operations for measures and streams.
+
+**Expression**: `sum(rate(banyandb_measure_total_written{job=~\"$job\", instance=~\"$instance\"}[$__rate_interval])) + sum(rate(banyandb_stream_tst_total_written{job=~\"$job\", instance=~\"$instance\"}[$__rate_interval]))`
```

Comment:
As you have been using PromQL, would you post the Grafana Dashboard configurations in the doc?